### ISSUE TITLE
Web component: Carousel - User navigates with tab/shift tab keys, focus on additional cards makes the keyboard navigation disorients

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4799]
## Detailed description

Issue: 
When additional cards are displayed on focus while the user navigates with tab/shift tab keys, it makes the keyboard navigation tedious and also disorients the Blind and low-vision users because it is not expected to trigger any action on tab/shift tab navigation

Expected Result:
Tab Focus should go to all the visible cards since all these are interactive elements. And the next set of cards should be displayed upon activation of previous or next button

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/1d0efbd2-cd72-4205-9a96-899d1bdb758b)
